package.json

```
@@ -3,15 +3,35 @@
   "private": true,
   "scripts": {
     "link": "lerna link",
-    "postinstall": "lerna bootstrap --no-ci && npm run build && npm run install-hooks",
+    "postinstall": "lerna bootstrap --no-ci && npm run build && npm run install-hooks && cd packages/mobile/ios && pod install",
```

Doesn't make a difference but i've been using this parameter

```suggestion
    "postinstall": "lerna bootstrap --no-ci && npm run build && npm run install-hooks && pod install --project-directory=ios",
```

---

this doesn't work because we need to be at least CD'd into packages/mobile, and at that point i'd say this is the same thing